daemon/analytics.c

```diff
@@ -974,14 +974,17 @@ void send_statistics(const char *action, const char *action_result, const char *
         action_result = "";
     if (!action_data)
         action_data = "";
+
+    const char *fail_reason = getenv("NETDATA_FAIL_REASON") ? getenv("NETDATA_FAIL_REASON") : "";
+
     char *command_to_run = mallocz(
         sizeof(char) * (strlen(action) + strlen(action_result) + strlen(action_data) + strlen(as_script) +
-                        analytics_data.data_length + (ANALYTICS_NO_OF_ITEMS * 3) + 15));
+                        analytics_data.data_length + (ANALYTICS_NO_OF_ITEMS * 3) + 15 + strlen(fail_reason) + 5));
```

Comment:
Why `+5` ?

Reply:
Rewritten to always have the field so the +5 for calculation for space, quotes and space is not needed